src/source-map.ts

```diff
@@ -1,6 +1,8 @@
+import * as Semver from 'semver';
 import type { Transformed } from './utils/transform/apply-transformers.js';
 
 const inlineSourceMapPrefix = '\n//# sourceMappingURL=data:application/json;base64,';
+const isPrepareStackTraceExposed = Semver.satisfies(process.versions.node, '>=20.12.0');
```

Comment:
Versioned features live here: https://github.com/privatenumber/tsx/blob/develop/src/utils/node-features.ts

Instead of `semver`, use the `isFeatureSupported` function